frontend/helpers/getConfig.js

```diff
@@ -0,0 +1,23 @@
+import { logger } from '@shopgate/pwa-core/helpers';
+
+let cached;
+
+/**
+ * Gets extension config or empty object on failure.
+ * @returns {Object}
+ */
+function getConfig() {
```

Comment:
as i already commented in in `components/SwitchHeader/index.jsx` i don't understand the reason for this helper. it doesn't do anything but introducing the need of additional function calls.

Reply:
done